main.js

```diff
@@ -160,4 +160,152 @@ const inventory = [
     originalStock: 10,
     sold: 8,
   },
-];
\ No newline at end of file
+];
+// Calculate TV's still to be sold
+function tvToBeSold(televisions) {
+  let count = 0
+  for (let i = 0; i < televisions.length; i++) {
+    const toBeSold = televisions[i].originalStock - televisions[i].sold
+    count = count + toBeSold
+  }
+  return count
+}
+//Set variable for TV's still to be sold
+const tvCount = tvToBeSold(inventory)
+//Get element on HTML and add TV's to be sold to counter
+const tvCountSite = document.getElementById('toBeSold')
+tvCountSite.textContent = tvCount.toString()
+//Array method for finding all TV names
+const allTvNames = inventory.map((television) => {
+  return television.name
+})
+//Array method for finding all sold out TV's
+const soldOutTv = inventory.filter((television) => {
+  if (television.sold === television.originalStock)
+    return true
+})
```

Comment:
`television.sold === television.originalStock` resulteert in een boolean. Als je dat resultaat terug wilt geven, dan is deze if overbodig. Dan kan hij als volgt:
```
const soldOutTv = inventory.filter(television => television.sold === television.originalStock);
```